asm2vec/binary_to_asm.py

```diff
@@ -52,7 +52,9 @@ def _fn_to_asm(pdf: dict | None, asm_minlen: int) -> str:
     """
     if pdf is None:
         return ''
-    if len(pdf['ops']) < asm_minlen:
+    if len(pdf.get('ops', [])) < asm_minlen:
+        if 'ops' not in pdf:
+            logging.info("KeyError: 'ops' not in pdf")
```

Comment:
The middle line is redundant, since you've done that check in the first:
Do you care if `ops` missing? or just avoid error?

```suggestion
    if len(pdf.get('ops', [])) < asm_minlen:
```

Reply:
OK, happy to keep if preferred